lib/api/redfish-1.0/task-service.js

```diff
@@ -0,0 +1,145 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+var injector = require('../../../index.js').injector;
+var redfish = injector.get('Http.Api.Services.Redfish');
+var waterline = injector.get('Services.Waterline');
+var Promise = injector.get('Promise');
+var _ = injector.get('_');
+var moment = require('moment');
+
+module.exports = {
+    taskServiceRoot: taskServiceRoot,
+    listTasks: listTasks,
+    getTask: getTask,
+    getSystemTasks: getSystemTasks
+};
+
+var dataFactory = function(identifier, dataName) {
+    var statusMap = {
+        'valid': 'Running',
+        'succeeded': 'Completed',
+        'cancelled': 'Killed',
+        'failed':  'Exception'
+    };
+
+    switch(dataName)  {
+        case 'graphObjects':
+            return Promise.map(waterline.graphobjects.find({}), function(graph) {
+                    return { 
+                        state: statusMap[graph._status],
+                        id: graph.id,
+                        status: 'OK',
+                        startTime: moment.utc(graph.createdAt).format(),
+                        endTime: (graph._status !== 'valid') ? 
+                            moment.utc(graph.updatedAt).format() : 
+                            '',
+                        name: graph.name,
+                        systemId: graph.node
+                    };
+                });
+    }
+};
+
+function taskServiceRoot(req, res) {
+    var options = {};
+    options.basepath = req.swagger.operation.api.basePath;
+    options.templateScope = ['global'];
+    options.url = req.url;
+    options.dateTime = moment.utc().format();
+    options.systems = [];
+
+    Promise.all([ dataFactory('', 'graphObjects') ])
+        .spread(function(graphs) {
+            _.forEach(graphs, function(graph) {
+                if (graph.systemId && -1 === options.systems.indexOf(graph.systemId))  {
+                    options.systems.push(graph.systemId);
+                }
+            });
+            options.graphs = graphs;
+            return redfish.render('redfish.1.0.0.taskservice.1.0.0.json', 
+                 'TaskService.1.0.0.json#/definitions/TaskService',
```

Comment:
it is really a conversation for @RackHD/corecommitters.  Moving the contents of templates/profiles out of the database and into the filesystem will impact the entire design.  It seems like a huge change to be made because you don't like the names, with little benefit to be had.

The contents are sourced from the database and not cached.  Update the database and stop reloading the app.  The new template takes affect immediately.